### DISCUSSION TITLE
SerialUPDI error

### DISCUSSION THREAD

**Discussion by sunny198828** [SpenceKonde/megaTinyCore#831] [Future Development Discussion]
hello,
I installed 2.6.1. when i burn the bootloader the error comes up:
Arduino IDE 1.8.3
Target: Attiny3227

Error: cannot parse fuse, 'invalid literal for int() with base 0: '{bootloader.CODESIZE}''

in ver. 2.5.11 is all ok.

Thanks for Help

**Comment by SpenceKonde** [comment]
This was fixed by 2.6.1
